### PR TITLE
schedule: set influence according to region size

### PR DESCRIPTION
**Pull request by rleungx**
# What problem does this PR solve? 
Currently, the store limit may influence the speed of merge. And some users hope to merge the region very fast.

### What is changed and how it works?
This PR eases this restriction by replacing `AddLearner` and `AddPeer` with `AddLightLearner` and `AddLightPeer` respectively. The behavior is just like before we introduce store limit.

**Update:**
**We still use `AddLearner` and `AddPeer`, but set the influence according to the region size.**

### Check List 

Tests 

 - Unit test

Related changes

 - Need to cherry-pick to the release branch
 - Need to be included in the release notes

### CODE REVIEW

**Comment by disksing** [comment]
Can update store limit increase the merge speed?

File: server/schedule/operator.go [hunk 1]
```diff
@@ -39,6 +39,10 @@ const (
 	RegionOperatorWaitTime = 10 * time.Minute
 	// RegionInfluence represents the influence of a operator step, which is used by ratelimit.
 	RegionInfluence int64 = 1000
+	// smallRegionInfluence represents the influence of a operator step, which is used by ratelimit.
```
Comment (shafreeck) [inline server/schedule/operator.go]: The comment here is the same as `RegionInfluence`, so what is the difference?

Comment (nolouch) [reply server/schedule/operator.go]: Do we need to consider the empty region?

Comment (rleungx) [reply server/schedule/operator.go]: Updated @shafreeck 
I think we don't need to consider the empty region. Once the region is empty (e.g. the size will be 1MB), the behavior is just like before we introduce the store limit. @nolouch

**Pull request merged**